win_installer.py

```diff
@@ -253,6 +253,10 @@ def run(bitness, repo_root):
         print('Installing mu with', venv_python)
         subprocess.run([venv_python, '-m', 'pip', 'install', repo_root])
 
+        print('Updating pip in the virtual environment', venv_python)
```

Comment:
I understand this works, but ideally `pip` would be updated right after creating the virtual environment, before the preceeding Mu installation lines.

Reply:
+1 I'll change this over... :-)